src/store/CrosshairStore.ts

```diff
@@ -236,7 +236,7 @@ class CrosshairStore {
                 dupMap.Open = dupMap.High = dupMap.Low = 1;
             }
         }
-        if (this.showSeries) {
+        if (this.showSeries && !this.showOhl) {
```

Comment:
Why have we added `!this.showOhl` check? 
OHLC chart type should also have the same date format

Reply:
Hi @balakrishna-deriv ! This is not related to date format, it is related to removing symbol field in chart info-box for non-ticks time intervals (please, check the notes in the card, this PR is common for 2 cards)